CHANGES.md

```diff
@@ -113,6 +113,18 @@
 
     - Removed the `Wrap` and `NextLayoutNoWrap` data constructors.
 
+  * `XMonad.Layout.IndependentScreens`
+
+    - Added `workspacesOn` for filtering workspaces on the current screen.
+
+    - Added `withScreen` to specify names for a given single screen.
+
+    - Added new aliases `PhysicalWindowSpace` and `VirtualWindowSpace`
+      for a `WindowSpace` for easier to read function signatures.
+
+    - Fixed a bug where `marshallPP` always sorted workspace names
+      lexically.
```

Comment:
This should be more prominent I think because while it does fix a bug with the Default ppSort, it also changes behaviour of marshallPP with non-default ppSorts.